code/modules/mob/living/simple_mob/subtypes/animal/space/alien.dm

```diff
@@ -2,7 +2,7 @@
 	name = "skathari soldier"
 	desc = "Terrible insects from beyond this galaxy!"
 	description_fluff = "Also know as the 'Bluespace Bugs', these insectoid invaders began to manifest suddenly around 2565 and their ravenous hunger for the very particles upon which modern FTL technology relies brought much of galactic society to a standstill. Their ravenous hunger for physical matter on the other hand, got people moving again fast - in the other direction!"
-	tt_desc = "X Extraneus Tarlevi"
+	tt_desc = "U Extraneus skathari"
```

Comment:
U? Is there precedent for U? X is the "unknown origin" designation used a couple of other places iirc

Reply:
the original PR where I did trinomial stuff specifies X for artificial species and U for species of unknown origin, and U is used for the funny space worm, though I know X is being used for a bunch of unknown origin stuff anyway since 
you know
the rules were never actually documented